pkg/apis/velero/v1/backup_storage_location.go

```diff
@@ -80,6 +80,11 @@ type BackupStorageLocationSpec struct {
 	// AccessMode defines the permissions for the backup storage location.
 	// +optional
 	AccessMode BackupStorageLocationAccessMode `json:"accessMode,omitempty"`
+
+	// BackupSyncPeriod defines how frequently to sync backup API objects from object storage.
+	// +optional
+	// +nullable
```

Comment:
Can we add a validation here?, e.g.

```
// +kubebuilder:validation:Minimum=0
```

Would that prevent negative values being set?

Reply:
I tried this, unfortunately looks like minimum validation only applies to integers -- build will otherwise fail with "must apply minimum to an integer". I agree though it would be great if we can validate this in the API spec.

Reply:
@prydonius I poked around kubebuilder to see how I can accomplish validation for this in the API spec and so far not found much, please let me know if you have another suggestion and I'll look into it.

Reply:
@betta1 ah sorry, I missed that this is not a basic integer field. I guess we could use a regex pattern with `+kubebuilder:validation:Pattern`, but I think that's going to be a little clunky. Was just a thought, we can just use the validation in the controller.

Reply:
Sounds good -- thanks for pointing me to kubebuilder, looks like a powerful tool for managing CRDs.